README.md

```diff
@@ -1 +1,28 @@
 # traces-backend-newrelic
+
+This is a New Relic backend implementation for the [Traces](https://github.com/socketry/traces) gem.
+
+
+## Quick start
+
+Add the Ruby agent to your project's Gemfile.
```

Comment:
```suggestion
Add the gem to your project's Gemfile. This will also install `newrelic_rpm`, the New Relic Ruby agent.
```
I'm not sure if "Ruby agent" is quite right here, but I don't know if "gem" is quite right either.